cmd/ddev/cmd/commands.go

```diff
@@ -35,8 +36,30 @@ func IsUserDefinedCustomCommand(cmd *cobra.Command) bool {
 // .ddev/commands/<servicename> and .ddev/commands/host
 // and if it finds them adds them to Cobra's commands.
 func addCustomCommands(rootCmd *cobra.Command) error {
+	// Custom commands are shell scripts - so we can't use them on windows without bash.
+	if runtime.GOOS == "windows" {
+		windowsBashPath := util.FindBashPath()
+		if windowsBashPath == "" {
+			fmt.Println("Unable to find bash.exe in PATH, not loading custom commands")
+			return nil
+		}
+	}
```

Comment:
This was further down, inside a loop for some reason - but it just needs to be run once and should be run early to avoid unnecessary processing